mapsforge-map-reader/src/main/java/org/mapsforge/map/reader/MapFile.java

```diff
@@ -27,7 +27,11 @@
 import org.mapsforge.core.util.LatLongUtils;
 import org.mapsforge.core.util.MercatorProjection;
 import org.mapsforge.core.util.Parameters;
-import org.mapsforge.map.datastore.*;
+import org.mapsforge.map.datastore.MapDataStore;
+import org.mapsforge.map.datastore.MapReadResult;
+import org.mapsforge.map.datastore.PoiWayBundle;
+import org.mapsforge.map.datastore.PointOfInterest;
+import org.mapsforge.map.datastore.Way;
```

Comment:
Can you revert this optimization of imports?
(as the code is unchanged to have clean history)